.atomist/apache2.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I was planning to use convention that maps from license type argument directly to file name rather then trying to handle apache-2 making an assumption that rug-rugs license has it